docs/src/router/drawer.tsx

```diff
@@ -141,7 +151,6 @@ export const NavigationDrawer: React.FC = () => {
                             '.BluiDrawerNavGroup-title': {
```

Comment:
Not sure, this gets applied, missing **&**

Reply:
This is working correctly.